client/lib/jetpack/types.ts

```diff
@@ -1,8 +1,8 @@
 import { ReactNode } from 'react';
-import type PageJS from 'page';
+import type { Context as PageContext } from 'page';
 import type Redux from 'redux';
 
-export interface Context extends PageJS.Context {
+export interface Context extends PageContext {
```

Comment:
Not for this PR, but I'd likely prefer to name `Context` a different way, like `WpcomContext` or `JetpackContext` in order to keep consistency with having the `Context` type resemble the same thing everywhere.